nrespipe/utils.py

```diff
@@ -570,3 +574,48 @@ def get_mag_from_simbad(target_name : str):
         mag = np.nan
 
     return mag
+
+
+def download_from_s3(frameid, output_directory):
+    url = f'{settings.ARCHIVE_FRAME_URL}/{frameid}'
+    response = requests.get(url, headers=settings.ARCHIVE_AUTH_TOKEN, stream=True).json()
+    with open(os.path.join(output_directory, response['filename']), 'wb') as f:
+        f.write(requests.get(response['url']).content)
+    return os.path.join(output_directory, response['filename'])
+
+
+def ingest_file(file_path):
+    retry = False
+    try_counter = 1
+    while retry:
+        try:
+            ingester.upload_file_and_ingest_to_archive(file_path)
```

Comment:
Is file_path a string? This function will need a file object.

Reply:
Good catch. Fixed. Thanks.